preprocessors/azure.py

```
@@ -0,0 +1,347 @@
+import pandas as pd
+import os
+from azureml.core import Dataset
+from tqdm import tqdm
+import hashlib
+from azureml.core import Workspace, Dataset, Datastore
```

Ruff (0.8.2)

`azureml.core.Workspace` imported but unused

Remove unused import

(F401)